Yoast/wordpress-seo
# Add a new page to the plugin that displays the onboarding wizard

Add a page for the Yoast SEO configuration wizard.

Testing
1. Go to this link and see if the wizard is loaded: /wp-admin/admin.php?page=wpseo_configurator
2. *If you want to test the wizard itself, you have to start Stubby.
3. Go to the other pages in the Yoast SEO settings and check if these load like they should.

If something doesn't work run: composer install, npm install/update and grunt build.
